### ISSUE TITLE
Packages Pane help buttons missing tooltips

### ISSUE THREAD

**Issue by nathanckuhn** [posit-dev/positron#15879]
<img width="282" height="67" alt="Image" src="https://github.com/user-attachments/assets/25fcf6b1-4056-4b55-bcfb-44a601557087" />

Type: <b>Bug</b>

In the Packages Pane, the buttons that open external and internal Help pages do not have a popup that appears on hover. Most other Positron buttons have a popup with the name for the button that appears when the cursor hovers over the button. I had to click on these buttons to find out what they do -- usually inadvisable. Please add!

Version: Positron 2026.08.2+4 (747dd536c5913adad71a568a57679bb2c5e5cd71, 2026-08-20T19:13:05Z)
Code - OSS version: VS Code 1.124.0
OS version: Windows_NT x64 10.0.26100
Modes:

<details><summary>Extensions (24)</summary>

Name|Identifier|Author|Version
---|---|---|---
Pastum|atsyplenkov.pastum|atsyplenkov|0.3.2
Ruff|charliermarsh.ruff|charliermarsh|2026.76.0
Path Intellisense|christian-kohler.path-intellisense|christian-kohler|2.8.0
Prettier - Code formatter|esbenp.prettier-vscode|esbenp|12.4.0
Outline Map|gerrnperl.outline-map|Gerrnperl|1.4.2
GitHub Pull Requests|github.vscode-pull-request-github|GitHub|0.156.0
Rainbow CSV|mechatroner.rainbow-csv|mechatroner|3.24.1
Render Line Endings|medo64.render-crlf|medo64|1.9.9
Pyrefly - Python Language Tooling|meta.pyrefly|meta|1.2.0
Git Graph|mhutchie.git-graph|mhutchie|1.30.0
Python Debugger|ms-python.debugpy|ms-python|2026.6.0
Jupyter|ms-toolsai.jupyter|ms-toolsai|2025.9.1
Jupyter Keymap|ms-toolsai.jupyter-keymap|ms-toolsai|1.1.2
Jupyter Notebook Renderers|ms-toolsai.jupyter-renderers|ms-toolsai|1.3.0
Jupyter Cell Tags|ms-toolsai.vscode-jupyter-cell-tags|ms-toolsai|0.1.9
Jupyter Slide Show|ms-toolsai.vscode-jupyter-slideshow|ms-toolsai|0.1.6
indent-rainbow|oderwat.indent-rainbow|oderwat|8.3.1
Material Icon Theme|pkief.material-icon-theme|PKief|5.38.1
Air - R Language Support|posit.air-vscode|Posit|0.28.0
Posit Assistant|posit.assistant|posit|1.2.0
Posit Publisher|posit.publisher|posit|2.13.1
Shiny|posit.shiny|Posit|1.4.2
Quarto|quarto.quarto|quarto|1.136.0
Code Spell Checker|streetsidesoftware.code-spell-checker|streetsidesoftware|4.7.5

</details>

**Comment by jmcphers** [comment]
Thanks for reporting, I can reproduce this!
